src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
```

Comment:
Not critical here, but would be great to follow the correct CSS properties order
https://9elements.com/css-rule-order/

Reply:
good evening! all bugs fixed👌